docs/development/extensions-contrib/k8s-jobs.md

```diff
@@ -217,59 +487,98 @@ data:
         druid.peon.mode=remote
         druid.indexer.task.encapsulatedTask=true
 ```
-#### Dynamic Pod Template Selection Config
-The Dynamic Pod Template Selection feature enhances the K8s extension by enabling more flexible and dynamic selection of pod templates based on task properties. This process is governed by the `PodTemplateSelectStrategy`. Below are the two strategies implemented:
+</details>
```

Comment:
```suggestion

</details>
```
I think there needs to be a newline in between HTML elements and Markdown for the code block to render properly. Otherwise GitHub shows it like this
![image](https://github.com/apache/druid/assets/7747997/bae0593c-dd72-4b1e-92e2-a484163a5ecd)

Reply:
It looks like the newline was required between the summary tag and the yaml block, but not here